packages/server/lib/util/escape_filename.ts

```diff
@@ -0,0 +1,17 @@
+import _ from 'lodash'
+
+const ampersandRe = /&/g
+const percentRe = /%/g
+const questionRe = /\?/g
+
+export function escapeFilenameInUrl (url: string) {
```

Comment:
Can't we use [encodeURI](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/encodeURI) instead of doing this ourselves?

Reply:
encodeURI doesn't encode `&` or `?`. 

`encodeURI('%&?.js') // returns '%25&?.js'`.

Reply:
Ah of course. Sorry, should have thought about that for more than a second.